test/run-tests.py

```diff
@@ -208,10 +208,14 @@ def SplitArgs(value):
 class CommandTemplate(object):
 
   def __init__(self, exe):
-    self.exe = exe
-    self.args = []
+    self.args = self._FixPythonExecutable(SplitArgs(exe))
     self.verbose_args = []
-    self.last_cmd = None
+
+  def _FixPythonExecutable(self, args):
+    exe, rest = args[0], args[1:]
+    if os.path.splitext(exe)[1] == '.py':
+      return [sys.executable, os.path.join(REPO_ROOT_DIR, exe)] + rest
+    return args
```

Comment:
This method doesn't use `self` so maybe make it a separate function like SplitArgs?